graphman-client/modules/graphman.js

```diff
@@ -43,18 +43,29 @@ module.exports = {
             method: 'POST',
             rejectUnauthorized: gateway.rejectUnauthorized,
             headers: headers,
-            auth: gateway.username + ":" + gateway.password,
             body: body || {}
         };
 
+        if (gateway.username && gateway.password) {
+            req.auth = gateway.username + ":" + gateway.password;
+        }
+        else if (gateway.keyFilename && gateway.certFilename) {
```

Comment:
please stick to the java or javascript coding style.

in this case, it should be:

`} else if (`

Reply:
Changed to java style